cmd/ddev/cmd/root.go

```diff
@@ -125,6 +125,7 @@ func Execute() {
 func init() {
 
 	RootCmd.PersistentFlags().BoolVarP(&output.JSONOutput, "json-output", "j", false, "If true, user-oriented output will be in JSON format.")
+	RootCmd.PersistentFlags().BoolVarP(&output.JSONOutput, "skip-hooks", "", false, "If true, any hook normally run by the command will be skipped.")
```

Comment:
Hi, @rfay I timidly started, I have a question before moving forward, initially I added the flag right next to `--json-output`, that' sensible to me, let me know otherwise, but the main question here is where to store the flag content. I tried to look but it doesn't look like there's any other flag similar to this or so scoped as json output.

Should this be on ddevapp? Anyway, probably best to share your insight on this first.

Reply:
My _guess_ is to create a new global var at around line 19:
```
var skipHooksArg bool
```

Reply:
I guess something like that would be the same as what we're doing of course with `--json-output`. I hate to do it with a global, even scoped, but don't have any other suggestions atm.